src/TimeoutConnector.php

```diff
@@ -30,14 +29,6 @@ public function __construct(int $timeout = self::DEFAULT_TIMEOUT)
      */
     public function connect(SqlConnectionConfig $config): Promise
     {
-        if (!$config instanceof ConnectionConfig) {
-            throw new \TypeError(\sprintf("Must provide an instance of %s to MySQL connectors", ConnectionConfig::class));
-        }
```

Comment:
This shouldn't have been dropped I guess?

Reply:
Yeah, I was a little overzealous with the delete key.